.editorconfig

```diff
@@ -14,3 +14,11 @@ trim_trailing_whitespace = true
 
 [*.md]
 trim_trailing_whitespace = false
+
+[*.java]
+indent_style = space
+indent_size = 4
+# Never use star imports
```

Comment:
Added this (IntelliJ-specific) configuration after I accidentally reset the import formatting style on `GenericContainer`. After this, we should converge towards consistent import style.